src/solvers/flattening/boolbv_quantifier.cpp

```diff
@@ -141,6 +141,16 @@ get_quantifier_var_max(const exprt &var_expr, const exprt &quantifier_expr)
 static optionalt<exprt>
 instantiate_quantifier(const quantifier_exprt &expr, const namespacet &ns)
 {
+  if(expr.variables().size() > 1)
+  {
+    // Qx,y.P(x,y) is the same as Qx.Qy.P(x,y)
+    auto new_variables = expr.variables();
+    new_variables.pop_back();
+    auto new_expression =
+      quantifier_exprt(expr.id(), expr.variables().back(), expr.where());
```

Comment:
So what about all the other variables, don't they just get dropped? Shouldn't `expr.where()` be replaced by `quantifier_exprt{expr.id(), new_variables, expr.where()}`?

Reply:
yes, fixed